kernel/traffic/core/src/test/java/org/apache/shardingsphere/traffic/algorithm/traffic/segment/SQLMatchTrafficAlgorithmTest.java

```diff
@@ -59,4 +60,10 @@ void assertMatchWhenNotExistSQLMatch() {
         assertFalse(sqlMatchAlgorithm.match(new SegmentTrafficValue(sqlStatement, "UPDATE `t_order` SET `order_id` = ?;")));
         assertFalse(sqlMatchAlgorithm.match(new SegmentTrafficValue(sqlStatement, "UPDATE `t_order_item` SET `order_id` = ? WHERE user_id = ?;")));
     }
+
+    @Test
+    void assertThrowExceptionWhenWithIllegalInit() {
```

Comment:
`assertInitWithIllegalProps()` is better.

Reply:
check